test/cli/suites-basic/dump.go

```diff
@@ -25,12 +25,14 @@ routes:
   - PUT
   name: route1
   service_id: svc1
+  status: 1
```

Comment:
Should we remove this field when dump configuration?

Reply:
I am not sure. Maybe users want to use dump to generate/update their configuration file? Keep these fields may avoid the diff of default values.